R/pkg/R/SQLContext.R

```diff
@@ -584,7 +584,7 @@ tableToDF <- function(tableName) {
 #'
 #' @param path The path of files to load
 #' @param source The name of external data source
-#' @param schema The data schema defined in structType
+#' @param schema The data schema defined in structType or a DDL-formatted string.
```

Comment:
hmm, to say `DDL` this might be too broad? what if someone pass the whole CREATE statement here?